### PR TITLE
Only scroll on pagination of tutorial cards if necessary

### PR DESCRIPTION
**Pull request by joekrill**
This should reduce some of the scrolling behavior for the paginated tutorial cards. They should now only scroll when the first card in the list is not visible in the viewport. So on mobile we should see the same behavior as before, but on desktop we won't scroll down when the first card is in the middle of the viewport.   

@srinjoyc let me know what you think about this behavior.  A further option if this is still not ideal would be to check if _any_ part of the card (or even the whole collection) is visible, and not scroll in those cases. 

#299

### CODE REVIEW

**Review by 10thfloor — APPROVED** [review]
Will merge once @srinjoyc approves.

**Comment by srinjoyc** [comment]
@joekrill - its a lot better but if we could, there are two more improvements we can make:
- The size of the cards should be the same - so when scrolling, we can avoid another jump on the first row of learn page
- If we can add some smooth scroll animation on pagination that might make it less jittery as well


Let me know if this is dooable/makes sense.

**Comment by schpet** [comment]
@srinjoyc FYI Joe and I discussed this, and decided it would be easier to get this merged and address your request in a follow up PR

**Pull request merged**
